src/itemset.js

```diff
@@ -115,11 +115,13 @@ define([
                     var possibleSrcs = _.map(datasources.getPossibleFixtures(), 
                                              function(val) { return val.src; }),
                         notCustom = _.contains(possibleSrcs, itemsetData.instance.src),
-                        choices = datasources.autocompleteChoices(itemsetData.instance.src);
+                        choices = datasources.autocompleteChoices(itemsetData.instance.src),
+                        strippedValue = itemsetData.valueRef.replace(/(\[.+])+/, ""),
+                        strippedLabel = itemsetData.labelRef.replace(/(\[.+])+/, "");
```

Comment:
I think I'm satisfied that this should typically be applied to some relative path like `attribute[filter][filter]`, and not `instance("foo")/foo/bar[a=1]/baz[b=2]`.

The regexp could be improved, although I'm not sure it matters.

Examples with with `/(\[.+])+/`
- `bar[a=1][b=2]` becomes `bar`
- `bar[a=1]/foo[b=2]` becomes `bar`
- `bar[a=1]/foo` becomes `bar/foo`

I think the first two are fine. Possibly more could be done on the second to validate that `bar` contains a `foo`, but probably not important right now. But the third one seems worse, if a bit far out.

Possible alternate: `/\[.+\].*/`
Matches from the first filter to the end of the string. Tries to match at least one well-formed filter, but there are malformed expressions that this will match too. Probably none of these crazy edge cases are worth caring about, although maybe we should decide how to err: do we want to strip possibly malformed filters, or do we want to try to strip well-formed filters only?